src/__tests__/SystemFields.cy.ts

```diff
@@ -0,0 +1,140 @@
+import SystemFields from '@components/smart/SystemFields.vue';
+import { mount } from 'cypress/vue';
+import { defineComponent } from 'vue';
+
+const status = 'Klad';
+const createdBy = 'Mr. Foo';
+const createdAt = '2023-05-24T13:45:03.427155+02:00';
+const updatedBy = 'Miss. Foo';
+const updatedAt = '2023-05-24T13:45:03.497579+02:00';
+
+describe('SystemFields - no data', () => {
+  const TestComponent = defineComponent({
+    components: { SystemFields },
+    template: '<SystemFields></SystemFields>',
+  });
+
+  it('renders', () => {
+    mount(TestComponent);
+  });
+
+  it('has an empty status field', () => {
+    mount(TestComponent);
+    cy.get('[data-cy="systemfield-status"]').should('have.text', 'Status: -');
+  });
+
+  it('has an empty aangemaakt door field', () => {
+    mount(TestComponent);
+    cy.get('[data-cy="systemfield-aangemaakt-door"]').should('have.text', 'Aangemaakt door: - op -');
+  });
+});
+
+describe('SystemFields - default data', () => {
+  const TestComponent = defineComponent({
```

Comment:
Ik krijg hier volgende eslint warning: `vue/one-component-per-file`
Misschien 1 describe voor `SystemFields` en dan een nested describe per story?
Of als deze manier de voorkeur heeft de eslint rule misschien disablen om een kleurboek aan warnings te vermijden :)